### PR TITLE
Some changes to AllChannel

### PR DESCRIPTION
**Pull request by LipuFei**

### CODE REVIEW

**Comment by LipuFei** [comment]
retest this please

File: Tribler/community/allchannel/community.py [hunk 1]
```diff
@@ -218,7 +217,7 @@ def create_channelcast(self):
                 # we've send something to this address, add to blocklist
                 self._blocklist[candidate] = now
 
-                nr_torrents = sum(len(torrent) for torrent in torrents.values())
+                nr_torrents = sum(len(infohash) for infohash in torrents.itervalues())
```
Comment (NielsZeilemaker) [inline Tribler/community/allchannel/community.py]: This is incorrect, the torrents dict acutally contains a list of infohashes per community identifier.
If you want to improve upon the previous version, rename torrent to community_torrents.

File: Tribler/community/allchannel/community.py [hunk 2]
```diff
@@ -248,11 +247,11 @@ def check_channelcast(self, messages):
     def on_channelcast(self, messages):
         for message in messages:
             toCollect = {}
-            for cid, torrents in message.payload.torrents.iteritems():
-                for infohash in self._selectTorrentsToCollect(cid, torrents):
+            for cid, infohashes in message.payload.torrents.iteritems():
```
Comment (NielsZeilemaker) [inline Tribler/community/allchannel/community.py]: Same here, and below

File: Tribler/community/allchannel/community.py (hunk 1 above)
Comment (LipuFei) [inline Tribler/community/allchannel/community.py]: Oh, made a mistake here. Should be `infohashes` instead. Thanks! @NielsZeilemaker

**Comment by LipuFei** [comment]
retest this please

**Comment by LipuFei** [comment]
retest this please

**Comment by LipuFei** [comment]
retest this please

**Comment by LipuFei** [comment]
retest this please

**Comment by LipuFei** [comment]
retest this please

**Comment by whirm** [comment]
@LipuFei ready?

**Comment by LipuFei** [comment]
@whirm ready

**Pull request merged**